bloom-housing/bloom
# fix: max update depth unit form issue

## Description

While testing #2661 , I noticed a `max update depth exceeded` warning caused by the unit form. This updates the form to get rid of that warning.

<img width="1197" alt="Screen Shot 2022-04-20 at 5 47 35 PM" src="https://user-images.githubusercontent.com/4401157/164350278-ecfff0ba-806c-477f-b490-891cd03dacb0.png">


## How Can This Be Tested/Reviewed?

In the partners portal go to add or edit a unit on the listing form. You should no longer see this warning.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
